src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -355,10 +362,10 @@
   that trapperkeeper will call on exit."
   PuppetDBServer
   [[:ConfigService get-config]
-   [:WebroutingService add-ring-handler get-route]]
+   [:WebroutingService add-ring-handler get-registered-endpoints]]
 
   (start [this context]
-         (start-puppetdb context (get-config) this add-ring-handler get-route))
+         (start-puppetdb context (get-config) this get-registered-endpoints))
```

Comment:
Any reason to pass the `fn` vs calling the `fn` here and passing the result to `start-puppetdb`?